credentials.md

```diff
@@ -0,0 +1,19 @@
+# Credential folder contents
+
+The folder includes a file commit-auth.js which contain sensitive information about the repository owner.
+
+Eg:
+
+## commit-auth.js
+
+```
```

Comment:
let the `./credentials/config.json` be like:
```json
{
  "git":{
    "url": "<git-url>",
    "username": "<git-username>",
    "password": "<git-password>"
}
```
hard-code all other data right into the program.